src/platform/macos.rs

```diff
@@ -33,6 +33,16 @@ pub trait WindowExtMacOS {
     /// space or taking control over the entire monitor.
     fn set_simple_fullscreen(&self, fullscreen: bool) -> bool;
 
+    /// Returns whether or not the window is occluded.
+    ///
+    /// Occlusion differs from window visibility as an occluded window
+    /// can be occluded from the screen surface for reasons such as being closed,
+    /// minimised, not set visible, or behind another window.
+    ///
+    /// Alternatively the event [`WindowEvent::Occluded`](crate::event::WindowEvent::Occluded)
+    /// can be used to track the occlusion state of the window.
```

Comment:
```suggestion
    /// Alternatively, the [`WindowEvent::Occluded`] event
    /// can be used to track the occlusion state of the window.
    ///
    /// [`WindowEvent::Occluded`]: crate::event::WindowEvent::Occluded
```
nit: grammar, inline link instead of reference link